src/Mapping/PropertyAccessors/PropertyAccessor.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ORM\Mapping\PropertyAccessors;
+
+/** @internal */
+interface PropertyAccessor
```

Comment:
I think a comment here would be good.

> a property accessor is a class that allows to read and write private/protected properties on objects
> we use them when … (during hydration?)
> there are several implementations because…